src/data/DateRangeSelector.ts

```diff
@@ -148,3 +149,74 @@ export function specialDateRangeToString(dateRange: SpecialDateRange): string {
       return '2022';
   }
 }
+
+export interface DateRangeRaw {
```

Comment:
Could you please explain the purpose of this type? Why are we not transforming directly to/from date range objects?